### PR TITLE
Lok 77

### PR DESCRIPTION
**Pull request by stein321**

### CODE REVIEW

**Comment by stein321** [comment]
Thinking about moving the shell scripts to their own directory

**Review by matthewpdias — APPROVED** [review]
looks good, should make some of the more complex oc commands harder to forget

File: createApps.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# create petclinic service
+oc new-app --image-stream=myproject/petclinic:latest --name=$APP_SVC_NAME
+oc expose svc/$APP_SVC_NAME
+oc set env --from=configmap/$APP_SVC_NAME-db-config dc/$APP_SVC_NAME
+oc set env --from=secret/$APP_SVC_NAME-secret dc/$APP_SVC_NAME
+
+# get hostname
+HOST_NAME=$(oc get route/$APP_SVC_NAME -o custom-columns=HOST:spec.host | tail -n +2)
+open http://$HOST_NAME/petclinic
```
Comment (ebracho) [inline createApps.sh:26]: Cool idea, that's convenient!

**Review by ebracho — APPROVED** [review]
LGTM

**Review by jbankes — APPROVED** [review]
LGTM

**Pull request merged**